### DISCUSSION TITLE
Submission of the tutorial

### DISCUSSION THREAD

**Discussion by MartinBilko** [irgroup-classrooms/dis17-2021#5] [Q&A] [answered]
How do you want us to confirm that we have done the tutorial?

In the PDF DIS17-01-introduction.pdf it says that we should take a screenshot. What should be on the screenshot?

**Comment by MartinBilko** [comment, marked as answer]
Answered by update of readme

**Reply by phschaer** [reply]
Actually, I planned to answer this question in the session tomorrow. :-)

But you are right. The details are now in the README.md file.

**Comment by Git-Romer** [comment]
Anywhere specific where we should upload our results?

**Reply by phschaer** [reply]
Excellent question! :-)

I added some additional information to the assignment description:

> ...
> 3. Make a screenshot of the results of a simple query from the official TREC-COVID topic set. We assigned one [topic number randomly to each student](topic-student.md). Name the file accordingly: `DIS17-2021-assignment1-LASTNAME-FIRSTNAME`.
> 4. Upload the screenshot until 2021-11-19 to this Sciebo folder: https://th-koeln.sciebo.de/s/FHerELroqNlwDPC

**Reply by Git-Romer** [reply]
Uploading only the screenshot or also our Student ID? (See [01 Introduction - Page 6](https://github.com/irgroup-classrooms/dis17-2021/blob/main/slides/DIS17-01-introduction.pdf))

**Reply by phschaer** [reply]
Nope, as we previously assigned each student her/his personal topic, you don't need to show your id.
